frontend/packages/kubevirt-plugin/src/models/templates/vm-yaml.ts

```diff
@@ -8,20 +8,25 @@ apiVersion: ${VirtualMachineModel.apiGroup}/${VirtualMachineModel.apiVersion}
 kind: ${VirtualMachineModel.kind}
 metadata:
   labels:
-    app: example
-    flavor.template.kubevirt.io/tiny: 'true'
+    app: vm-example
+    flavor.template.kubevirt.io/Custom: 'true'
     os.template.kubevirt.io/fedora31: 'true'
-    vm.kubevirt.io/template.revision: '1'
```

Comment:
same here: this file is generated from the code above, but removed the common templates labels and changed flavor to `Custom`